src/arrayReverse.test.js

```diff
@@ -11,10 +11,15 @@ describe(`Function 'arrayReverse':`, () => {
 
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
+  it(`should return a reverse text`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
-  // write more tests here
+  it('should return a reversed word', () => {
+    expect(arrayReverse(['Hello'])).toEqual(['olleH']);
+  });
+
+  it('should return a reversed word', () => {
```

Comment:
The description for this test case is identical to the one on line 18. Consider updating it to better describe the specific test case, such as 'should return a reversed word with special characters'.